src/AWS.Deploy.Recipes.CDK.Common/CDKRecipeCustomizer.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AWS.Deploy.Recipes.CDK.Common
+{
+    public delegate void CustomizePropsDelegate<GeneratedConstruct>(CustomizePropsEventArgs<GeneratedConstruct> evnt);
```

Comment:
nit: might as well use "event" instead of "evnt"

Reply:
`evnt` is the common name to use since `event` is a reserved word in .NET.